cogs/logging.py

```diff
@@ -37,13 +54,64 @@ async def on_message_delete(self,message):
     # TODO implement edit messages
     @commands.Cog.listener()
     async def on_message_edit(self,before,after):
-        pass
+        guild_id = before.guild.id
+        channel = self.logging_channels[guild_id]
+
+        before_string = ("**Before:**" if len(before.content) <= 750
+                        else "**Before:** (Trimmed at 750)")
+        after_string = ("**After:**" if len(after.content) <= 750
+                        else "**After:** (Trimmed at 750)")
+
+        before_sanitized_content = before.content[:750].replace("```", "‵‵‵") 
+        after_sanitized_content = after.content[:750].replace("```", "‵‵‵") 
+
+
+        payload = (f"# Message edited\n"
+                        f"**Member:** `{before.author.id}` <@{before.author.id}>\n"
+                        f"{before_string} ```{before_sanitized_content}```\n"
+                        f"{after_string} ```{after_sanitized_content}```")
+        
+        await channel.send(
+                    content=payload   
+                    )
 
 
     # TODO implement purge messages
     @commands.Cog.listener()
     async def on_raw_bulk_message_delete(self, payload):
-        pass
+        '''
+        Need a way to purge messages before this is fully implemented.
```

Comment:
This will remain a TODO of sorts, which is fine, but not sure if we should merge this part yet at all if it's not implemented.

Reply:
Agreed, yeah. Will talk a bit more with the moderation team to see when purging messages will be relevant for us, as well.